routes/web.php

```diff
@@ -23,3 +23,11 @@
 
 Route::post('/employer/register', 'RegisterEmployerController@register')
 ->name('employer.register');
+
+
+
+
+Route::post('/employer/update/profile', 'EmployerUpdateProfile@update')->name('employer.update');
```

Comment:
```suggestion
Route::post('/employer/update/profile', 'EmployerUpdateProfile@update')->name('employer.update');
```
Thừa dòng nhiều quá.